icons/captions.json

```
@@ -0,0 +1,18 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "defaultlp"
+  ],
+  "tags": [
+    "closed captions",
+	  "subtitles",
+	  "subhead",
+	  "transcription",
+	  "transcribe",
+	  "dialogue",
+    "accessibility"
+  ],
+  "categories": [
+    "multimedia"
+  ]
```

We should remove the `subtitles` icon and add it as an alias.
```suggestion
  ],
  "aliases": [
    "subtitles"
  ]
```